multus/multus.go

```diff
@@ -34,8 +35,16 @@ import (
 	k8s "github.com/intel/multus-cni/k8sclient"
 	"github.com/intel/multus-cni/types"
 	"github.com/vishvananda/netlink"
+	"k8s.io/apimachinery/pkg/util/wait"
 )
 
+var defaultReadinessBackoff = wait.Backoff{
+	Steps:    4,
+	Duration: 10 * time.Millisecond,
+	Factor:   5.0,
```

Comment:
Idea is to set 250 millis, and factor of 4, for .25 seconds, 1 second, 4 second, 16 secons....